jsonrpc/src/methods/beacon.json

```diff
@@ -200,5 +200,17 @@
     "result": {
       "$ref": "#/components/contentDescriptors/GossipResult"
     }
+  },
+  {
+    "name": "portal_beaconGetContent",
+    "summary": "Get content from the local database if it exists, otherwise look up the target content key in the network.",
+    "params": [
+      {
+        "$ref": "#/components/contentDescriptors/ContentKey"
+      }
+    ],
+    "result": {
```

Comment:
This method will also require a ContentNotFoundError, see PR: https://github.com/ethereum/portal-network-specs/pull/343.

Reply:
Sure, I'll add that. Thanks